test/TestUtils/RepoDirectoriesProvider.cs

```diff
@@ -28,28 +29,40 @@ public class RepoDirectoriesProvider
         public string NugetPackages => _nugetPackages;
         public string CorehostPackages => _corehostPackages;
         public string CorehostDummyPackages => _corehostDummyPackages;
+        public string DotnetSDK => _dotnetSDK;
 
         public RepoDirectoriesProvider(
             string repoRoot = null,
             string artifacts = null,
             string builtDotnet = null,
             string nugetPackages = null,
             string corehostPackages = null,
-            string corehostDummyPackages = null)
+            string corehostDummyPackages = null,
+            string dotnetSdk = null)
         {
-            _repoRoot = repoRoot ?? Path.Combine(Directory.GetCurrentDirectory(), "..", "..");
-            
-            string baseArtifactsFolder = artifacts ?? Path.Combine(_repoRoot, "artifacts");
+            _repoRoot = repoRoot ?? Directory.GetParent((Directory.GetCurrentDirectory())).Parent.FullName;
+
+            string baseArtifactsFolder = artifacts ?? Path.Combine(_repoRoot, "bin");
```

Comment:
Should `Artifacts` be renamed here and everywhere now that this folder is no longer named "artifacts"?